lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -739,18 +745,22 @@ selectTxs txids = do
 selectTxHistory
     :: forall t. PersistTx t
     => W.WalletId
+    -> W.SortOrder
     -> [Filter TxMeta]
     -> SqlPersistT IO [(W.Hash "Tx", (W.Tx t, W.TxMeta))]
-selectTxHistory wid conditions = do
+selectTxHistory wid dir conditions = do
```

Comment:
Just a small thing, but for a value of type `SortOrder`, perhaps it would be clear to call it `order`, or `sortOrder`?

This way, names of types and values are consistent.

(I can understand why this might have happened, as we introduced `SortOrder` and `SortDirection` at the same time.)

Reply:
💯 thanks for bringing up!